dmarc.tex

```diff
@@ -0,0 +1,66 @@
+\section{Data Management Conceptual Architecture} \label{sect:dmarc}
+
+In operations components of DM are installed and running in several locations. Namely:
+\begin{itemize}
+\item Commissioning cluster (which may be physically in NCSA)
```

Comment:
Chuck wanted it at the Base so that it's not dependent on the international link and to keep latency down for interactive use.